typesystem/fields.py

```diff
@@ -310,7 +310,7 @@ class Decimal(Number):
     numeric_type = decimal.Decimal
 
     def serialize(self, obj: typing.Any) -> typing.Any:
-        return float(obj)
+        return float(obj) if obj else None
```

Comment:
That doeesn't look right to me for the `0` case.

Reply:
Probably `return None if obj is None else float(obj)`, right?

Reply:
Thirth time on the same week with the same mistake... haha, seems like a complex relation with python conditional booleans :)